### PR TITLE
Hunter skill - Memo (WIP)

### PR DESCRIPTION
**Pull request by A-Powell**
Added the hunter skill with 10 creatures to hunt, All with correct loots, exp, and id's.
Added only 2 initial hunting equipment (currently not in use)
Re-spawn timers may need to be tweaked.
Tested, linted, Everything is working as intended

### CODE REVIEW

File: src/commands/Minion/hunt.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+					timetoHunt = Math.floor(timetoHunt * ((100 - supply.reductionPercent) / 100));
+					boosts.push(`${supply.reductionPercent}% for ${itemNameFromID(supply.id)}`);
+					break;
+				}
+			}
+		}
+
+		// If no quantity provided, set it to the max.
+		if (quantity === null) {
+			quantity = Math.floor((Time.Minute * 30) / timetoHunt);
```
Comment (gc) [inline src/commands/Minion/hunt.ts:94]: any occurrences of Time.Minute*30 have to be changed to the new system (see code in the bot)

File: src/commands/Minion/hunt.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		let timetoHunt = determineScaledCreatureTime(
+			creature!.xp,
+			creature.respawnTime,
+			msg.author.skillLevel(SkillsEnum.Hunter)
+		);
+
+		// If the user has supplies, apply boost
+		const bank = msg.author.settings.get(UserSettings.Bank);
+		const boosts = [];
+		if (msg.author.skillLevel(SkillsEnum.Hunter) >= 61) {
```
Comment (gc) [inline src/commands/Minion/hunt.ts:82]: why do items that are required to do hunter give a small boost, whats the significance of level 61?

File: src/tasks/minions/hunterActivity.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+			user.minionName
+		} asks if you'd like them to do another of the same trip.`;
+
+		if (newLevel > currentLevel) {
+			str += `\n\n${user.minionName}'s Hunter level is now ${newLevel}!`;
+		}
+
+		const loot: { [key: string]: number } = {};
+
+		Creature.drops?.forEach(drop => (loot[itemID(drop.toString())] = quantity));
```
Comment (gc) [inline src/tasks/minions/hunterActivity.ts:38]: `for of` instead of forEach

File: src/commands/Minion/hunt.ts (hunk 2 above)
Comment (A-Powell) [reply src/commands/Minion/hunt.ts:82]: This is more of a placeholder for the moment, I think some certain items give boosts, not these in particular, will be changed!

**Comment by Lajnux** [comment]
Hunter been added since a while

**Pull request closed without merging**